include/cling/Interpreter/InvocationOptions.h

```diff
@@ -32,12 +46,16 @@ namespace cling {
     std::vector<std::string> LibsToLoad;
     std::vector<std::string> LibSearchPath;
     std::vector<std::string> Inputs;
+    cling::CompilerOpts CompilerOpts;
```

Comment:
We use the notion `cling::CompilerOptions CompilerOpts`, i.e. we shorten the variable name and its accessors (if any) and we keep the class name verbose.